Sources/SwiftProtobuf/Google_Protobuf_Struct.swift

```diff
@@ -407,69 +410,67 @@ public struct Google_Protobuf_Value: Message, Proto3Message, _MessageImplementat
             if let newValue = newValue {
                 kind = .listValue(newValue)
             } else {
-                kind = .None
+                kind = nil
             }
         }
     }
 
-    public enum OneOf_Kind: ExpressibleByNilLiteral, OneofEnum {
+    public enum OneOf_Kind: OneofEnum {
         case nullValue(Google_Protobuf_NullValue)
         case numberValue(Double)
         case stringValue(String)
         case boolValue(Bool)
         case structValue(Google_Protobuf_Struct)
         case listValue(Google_Protobuf_ListValue)
-        case None
-
-        public init(nilLiteral: ()) {
-            self = .None
-        }
-
-        public init() {
```

Comment:
No, it doesn't make sense for a `oneof` case property to be initialized without *some* field set now that they're optional.